cylc/flow/workflow_db_mgr.py

```diff
@@ -632,13 +632,13 @@ def put_update_task_outputs(self, itask):
         """Put UPDATE statement for task_outputs table."""
         set_args = {
             "outputs": json.dumps(
-                list(itask.state.outputs.iter_completed_messages())
+                itask.state.outputs.get_completed_outputs()
```

Comment:
List changed to dictionary.

This looks like a https://github.com/cylc/cylc-flow/labels/db%20change that has not been handled properly. Is it actually needed?

Reply:
(I need to check this)